codeclimate_test_reporter/components/file_coverage.py

```diff
@@ -18,6 +22,7 @@ def payload(self):
         }
 
     def __process(self):
+        self.file_body = open(self.__filename(), "r", encoding="utf-8-sig").read()
```

Comment:
Are there any paths through `__process` that wouldn't have to read the file? I'm wondering if making a method to read the file (with proper encoding) would be better than eager-reading it and stashing content on `self` like this.